### PR TITLE
Initial version of web site

### PR DESCRIPTION
**Pull request by frreiss**
This PR includes an initial version of the Github Pages web site for the project. 

Here's a live copy, running off of my branch: https://frreiss.github.io/tep-fred/

### CODE REVIEW

File: docs/README.md [hunk 1]
```diff
@@ -1,9 +1,132 @@
...
 
-This should be the main page of the Github Pages site.
+Make NLP data easier to understand with Pandas DataFrames.
+
+![Make NLP data easier to understand with Pandas](images/before_after.png)
+
+Text Extensions for Pandas lets you use Pandas DataFrames for representing and manipulating the complex data structures that flow through
+modern natural language processing (NLP) applications.
+
+We provide **Pandas extension types** to cover common data structures for representing natural language data, plus **library integrations** that turn the  coutputs of popular NLP libraries into easy-to-understand DataFrames.
```
Comment (kmh4321) [inline docs/README.md:12]:
```suggestion
We provide **Pandas extension types** to cover common data structures for representing natural language data, plus **library integrations** that turn the outputs of popular NLP libraries into easy-to-understand DataFrames.
```

File: docs/README.md [hunk 2]
```diff
@@ -1,9 +1,132 @@
...
+
+Want to try it out? Just type
```
Comment (kmh4321) [inline docs/README.md:14]:
```suggestion
Want to try it out? Just type the following command in your terminal:
```

File: docs/README.md [hunk 3]
```diff
@@ -1,9 +1,132 @@
...
+
+# Extension Types
+
+We extend the [Pandas data model](https://pandas.pydata.org/pandas-docs/stable/user_guide/basics.html#basics-dtypes) with two new concepts: **Spans** and **Tensors**
+
+We use the Pandas [extensions API](https://pandas.pydata.org/pandas-docs/stable/development/extending.html#extension-types) to integrate our NLP data types tightly with the rest of Pandas DataFrames.
+
+## Spans and SpanArrays
+
+A *span* is a range of charactors or tokens within a document. Our
```
Comment (kmh4321) [inline docs/README.md:32]:
```suggestion
A *span* is a range of characters or tokens within a document. Our
```

**Review by kmh4321 — CHANGES_REQUESTED** [review]
minor typos and rewording, but overall looks great!

File: docs/README.md (hunk 2 above)
Comment (BryanCutler) [reply docs/README.md:14]: What about "Want to try it out? Use the following command to install:"

**Review by BryanCutler — APPROVED** [review]
LGTM, just one minor suggestion

File: docs/README.md (hunk 1 above)
Comment (frreiss) [reply docs/README.md:12]: Fixed now.

Comment (frreiss) [reply docs/README.md]: Fixed now.

File: docs/README.md (hunk 3 above)
Comment (frreiss) [reply docs/README.md]: Fixed now.

File: docs/README.md (hunk 2 above)
Comment (frreiss) [reply docs/README.md]: Fixed now.

**Pull request merged**